types/zengin-code/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for zengin-code 1.0
+// Project: https://github.com/zengin-code/zengin-js#readme
+// Definitions by: Ken Takahashi <https://github.com/redimpulz>
```

Comment:
This username should match yours (unless there's a good reason otherwise)

Reply:
@RyanCavanaugh Thank you. I fixed.